certora/specs/SNTTokenController.spec

```diff
@@ -0,0 +1,3 @@
+rule sanity {
+  assert true;
+}
```

Comment:
This is just a placeholder rule to ensure the prover is happy.
I'm not sure if there's any additional rules to add here at this point, given that SNT v2 is largely an instance of `MiniMeToken`